### ISSUE TITLE
2021-04-11 23:21:29|0.6.2|ERROR|'Select Shipping Options - AmazonSmile Checkout' is not a known page title. Please create issue indicating the title with a screenshot of page

### ISSUE THREAD

**Issue by dkrovich** [Hari-Nagarajan/fairgame#669]
Using an Amazon business account so it just may not be supported by fairgame but filing anyway just in case.  Fairgame works as expected on a standard account.  

Can get through select shipping address with --shipping-bypass but then I get stuck at the following page.  Seems like it is expecting a place order screen but gets this instead.  

![Screenshot from 2021-04-11 23-18-04](https://user-images.githubusercontent.com/4078900/114336705-8739c880-9b1d-11eb-854c-4b0eb15d4c72.png)

**Comment by DakkJaniels** [comment]
Yeah seems like business accounts have an extra page in there now. Maybe someone will submit a fix, but since just using a regular account fixes this issue, it's not a very high priority.

**Comment by dkrovich** [comment]
Ok.  Do you think it would be a lot of work to add?  I may take a look at it.  I haven't looked at the code but perhaps a change like this would be somewhat simple: like copy a page, change a few values, and profit?  Or more to it?

**Comment by DakkJaniels** [comment]
Probably not a ton of work, go for it.

**Comment by nsar1907** [comment]
i am having the same error for amazon.de

**Comment by DakkJaniels** [comment]
> i am having the same error for amazon.de

Then resolve it in the same way I described

**Comment by nsar1907** [comment]
> > i am having the same error for amazon.de
> 
> Edit - sorry, thought this was a different issue.

does auto ordering work for you at the moment ?

**Comment by nsar1907** [comment]
> Yeah seems like business accounts have an extra page in there now. Maybe someone will submit a fix, but since just using a regular account fixes this issue, it's not a very high priority.

my account is not bussness one and having the problem for de and fr now, amazon it works as expected

**Comment by DKnightX91** [comment]
I been getting this lately aswell, any workaround?

**Comment by RadianceNL** [comment]
Same here with a regular account on Amazon.de

**Comment by VIncentOOF** [comment]
me too when i do that amazon smile that happened too and I was using my personal account.
